api/v1beta2/flowcollector_types.go

```diff
@@ -351,6 +351,11 @@ type FLPMetrics struct {
 	// `namespace_ingress_packets_total`, `namespace_flows_total`, `node_egress_bytes_total`, `node_egress_packets_total`,
 	// `node_ingress_bytes_total`, `node_ingress_packets_total`, `node_flows_total`, `workload_egress_bytes_total`,
 	// `workload_egress_packets_total`, `workload_ingress_bytes_total`, `workload_ingress_packets_total`, `workload_flows_total`.
+	// When the `PacketDrop` feature is enabled in `spec.agent.ebpf.features`, these additional metrics are available:
```

Comment:
@jotak This tooltip needs to be updated to include the privileged setting too for PacketDrop

Reply:
And is there any improvement targeted for the same tool-tip since it looks very cramped in a small space and tough to read all the listed metrics

Reply:
Other than tool-tip, LGTM

Reply:
So I can add some line breaks which work in the docs, but if you're referring to the OLM console I fear it will make it even worse:

![image](https://github.com/netobserv/network-observability-operator/assets/2153442/0893c07b-5229-41b4-bccd-3d71bc8cf2f7)

@jpinsonneau is that something we have control on, when customizing the OLM form? Can we set line breaks?

Another option could be to not list available metrics here and put an external link to the doc instead.

Reply:
in last commit I reduced size of this doc and externalized for more info:

![image](https://github.com/netobserv/network-observability-operator/assets/2153442/9e52e015-1815-44f0-8861-0db5c5e63dfe)

Reply:
the external link will show this: https://github.com/netobserv/network-observability-operator/blob/6c7662edbfc03f9b26f69987b9db3c4e61254af4/docs/Metrics.md

Reply:
yeah I think this looks cleaner. Let me test it out just in case to see how it looks on a cluster

Reply:
Looks better now on console